### PR TITLE
ISAICP-5018: Paragraphs accordion item.

### PR DESCRIPTION
**Pull request by idimopoulos**

### CODE REVIEW

File: tests/features/paragraphs/custom_page.paragraphs.feature [hunk 1]
```diff
@@ -71,3 +71,46 @@ Feature:
     Then I should see the success message "Custom page Don't Mess with the Zohan has been updated."
     And I should see "I'm half Australian, half Mt. Everest"
     And the response should contain "{\"foo\":\"bar\"}"
+
+  @javascript
+  Scenario: Add an accordion to the custom page.
+    Given I am logged in as a moderator
```
Comment (pfrenssen) [inline tests/features/paragraphs/custom_page.paragraphs.feature:77]: Wouldn't it be better to use the facilitator as the editor of the custom page? I don't think it is a common task for moderators to create content in collections.

File: tests/features/paragraphs/custom_page.paragraphs.feature [hunk 2]
```diff
@@ -71,3 +71,46 @@ Feature:
...
+    Then I press "Add Accordion" in the "Custom page body" paragraphs field
+
+    # 3 paragraphs are the default number when adding an accordion.
+    # 1. the accordion wrapper, 2. accordion item and 3. the simple paragraph contained in the item.
+    And there should be 3 paragraph in the "Custom page body" field
+    When I press "Add Accordion item"
+    # An accordion item and a simple paragraph have been added.
+    Then there should be 5 paragraphs in the "Custom page body" field
+
+    Given I fill in the 1st "Item label" with "Super" in the "Custom page body" field
```
Comment (pfrenssen) [inline tests/features/paragraphs/custom_page.paragraphs.feature:96]: I like how this is so nicely human readable :+1:

File: tests/src/Traits/UtilityTrait.php [hunk 3]
```diff
@@ -181,4 +181,24 @@ protected function waitUntil(callable $callback, int $timeout = 5) {
...
+   * @param string $number
+   *   The ordinal number.
+   *
+   * @return int
+   *   The number.
+   *
+   * @throws \Exception
+   *   Thrown if the number is not an ordinal.
+   */
+  protected function convertOrdinalToNumber(string $number): int {
```
Comment (pfrenssen) [inline tests/src/Traits/UtilityTrait.php:196]: Here's the magic :+1:

**Review by pfrenssen — CHANGES_REQUESTED** [review]
Just a small remark, let's test with the facilitator rather than the moderator, the facilitator is a better showcase for this feature.

File: tests/src/Traits/UtilityTrait.php (hunk 3 above)
Comment (idimopoulos) [reply tests/src/Traits/UtilityTrait.php:196]: I got that from @brummbar  :D

Comment (pfrenssen) [reply tests/src/Traits/UtilityTrait.php:196]: :bear:

File: web/modules/custom/collection/config/install/og.og_role.rdf_entity-collection-facilitator.yml [hunk 4]
```diff
@@ -34,13 +36,16 @@ permissions:
...
+  - 'delete paragraph content accordion'
+  - 'delete paragraph content accordion_item'
   - 'delete paragraph content simple_paragraph'
   - 'edit any custom_page content'
   - 'edit any discussion content'
   - 'edit any document content'
   - 'edit any event content'
   - 'edit any glossary content'
   - 'edit any news content'
+  - 'edit any tallinn_report content'
```
Comment (pfrenssen) [inline web/modules/custom/collection/config/install/og.og_role.rdf_entity-collection-facilitator.yml:48]: This seems unrelated to this ticket.

**Review by pfrenssen — COMMENTED** [review]
Is it intentional to give a permission to facilitators to edit Tallinn content?

**Review by pfrenssen — APPROVED** [review]
Looking good!

**Pull request merged**